rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/report/SW360ReportController.java

```diff
@@ -103,12 +105,15 @@ public void getProjectReport(
             @RequestParam(value = "variant", required = false) String variant,
             @Parameter(description = "Template for generating report. Can be supplied with modules [" + LICENSE_INFO + ", " + EXPORT_CREATE_PROJ_CLEARING_REPORT + "]")
             @RequestParam(value = "template", required = false, defaultValue = "") String template,
-            @Parameter(description = "The external Ids of the project. Can be supplied with modules [" + LICENSE_INFO + ", " + EXPORT_CREATE_PROJ_CLEARING_REPORT + "]", example = "376577")
+            @Parameter(description = "The external Ids of the project. External Ids can be supplied multiple with comma separated. " +
+                    "Can be supplied with modules [" + LICENSE_INFO + ", " + EXPORT_CREATE_PROJ_CLEARING_REPORT + "]", example = "376577")
             @RequestParam(value = "externalIds", required = false, defaultValue = "") String externalIds,
             @Parameter(description = "Generate report for only current project or with Sub projects. Can be supplied with modules [" + LICENSE_INFO + ", " + EXPORT_CREATE_PROJ_CLEARING_REPORT + "]")
             @RequestParam(value = "withSubProject", required = false, defaultValue = "false") boolean withSubProject,
             @Parameter(description = "Type of SBOM file", schema = @Schema(allowableValues = {XML_FILE_EXTENSION, JSON_FILE_EXTENSION}))
             @RequestParam(value = "bomType", required = false) String bomType,
+            @Parameter(description = "Selected release relationships.")
```

Comment:
As with other parameters, please tell which modules use this (`LICENSE_INFO`, `EXPORT_CREATE_PROJ_CLEARING_REPORT`)

Reply:
Also, please document the type of values which can go here, maybe provide an example, etc.